README.md

```diff
@@ -17,13 +17,13 @@ Click on the name of a plugin or module to view that content's documentation:
 ### Roles
 Name | Description
 --- | ---
-[cloud.azure_roles.load_balancer](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/load_balancer/README.md)|A role to manage Azure Load Balancer.
-[cloud.azure_roles.managed_postgresql](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/managed_postgresql/README.md)|A role to manage Azure PostGreSQL Database.
-[cloud.azure_roles.network_interface](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/network_interface/README.md)|A role to manage Azure Network Interface.
-[cloud.azure_roles.networking_stack](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/networking_stack/README.md)|A role to manage Azure Networking Stack.
-[cloud.azure_roles.resource_group](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/resource_group/README.md)|A role to manage Azure Resource Group.
-[cloud.azure_roles.security_group](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/security_group/README.md)|A role to manage Azure Security Group.
-[cloud.azure_roles.virtual_machine](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/virtual_machine/README.md)|A role to manage Azure Virtual Machine.
+[cloud.azure_roles.azureconfig_load_balancer](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/azureconfig_load_balancer/README.md)|A role to manage Azure Load Balancer.
+[cloud.azure_roles.azure_managed_postgresql](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/azure_managed_postgresql/README.md)|A role to manage Azure PostGreSQL Database.
+[cloud.azure_roles.azureconfig_network_interface](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/azureconfig_network_interface/README.md)|A role to manage Azure Network Interface.
+[cloud.azure_roles.azureconfig_networking_stack](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/azureconfig_networking_stack/README.md)|A role to manage Azure Networking Stack.
+[cloud.azure_roles.azure_resource_group](https://github.com/ansible-collections/cloud.azure_roles/blob/main/roles/azure_resource_group/README.md)|A role to manage Azure Resource Group.
```

Comment:
What is the difference between the roles called azure_something and azure_managed_something and azureconfig_something? Not sure, but probably adding a more accurate description for each role would also help.

Reply:
The intend was to align with what has been done for AWS roles, but I think it is better to keep it simple
Just name all the role azure_xxx

Reply:
The rename should result in the roles being named after their use cases.  For example, we have cloud.aws_ops.customized_ami for the role that allows a user to create a custom EC2 AMI.  We didn't name this role cloud.aws_ops.ami because it only manages the specific function of customization.  

Also please ensure the FQCN's use the new agreed repo name (from the Jira ticket) of cloud.azure_ops - the repo will be renamed on Github once this PR is merged (right before it is migrated to the https://github.com/redhat-cop org).

Reply:
could you please have a look and see if it is fine now with the naming? If not could you please suggest names? Regards